docker-images/docker_build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+########################
+# Purpose: Build the required docker images for installation
+# Author: Ankshit Jain
+# Date: 23-Nov-2017
+# Invocation: $sudo bash docker_build.sh
+# Dependencies: This script requires docker to be already installed on the system.
+########################
+
+cd ../main_server/ || exit
+sudo docker build -t main_server .  | tee main_server_container_build.log
```

Comment:
Is it better to move these logs to log directory available at the relative location "../log"?